src/CalendarApp.Domain/Services/MeetingRemoverService.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CalendarApp.Contracts;
+using CalendarApp.DataAccess.Repositories.Interfaces;
+using CalendarApp.Domain.Exceptions;
+using CalendarApp.Domain.Services.Interfaces;
+
+namespace CalendarApp.Domain.Services;
+
+public class MeetingRemoverService : IMeetingRemoverService
+{
+    public string DeleteMeeting(string meetingName)
```

Comment:
it's bad style for delete return string, better return number (number of meetings removed) and create string in presenter based on this information